geocoder_tester/world/poi/test_airports.csv

```diff
@@ -0,0 +1,7185 @@
+Örebro Airport;4;Sweden;59.22370147705078,15.038000106811523,1500;OurAirports;ORB
+Stockholm Västerås Airport;4;Sweden;59.58940124511719,16.63360023498535,1500;OurAirports;VST
+Luleå Airport;4;Sweden;65.543800354004,22.121999740601,1500;OurAirports;LLA
+Vidsel Air Base;4;Sweden;65.87529754638672,20.149900436401367,1500;OurAirports;\N
+Arboga Airport;4;Sweden;59.386600494384766,15.924099922180176,1500;OurAirports;\N
+Stockholm-Arlanda Airport;4;Sweden;59.651901245117,17.918600082397,1500;OurAirports;ARN
+Stockholm-Bromma Airport;4;Sweden;59.354400634765625,17.941699981689453,1500;OurAirports;BMA
+Borlange Airport;4;Sweden;60.422000885009766,15.515199661254883,1500;OurAirports;BLE
+Hultsfred Airport;4;Sweden;57.525798797607,15.823300361633,1500;OurAirports;HLF
+Gävle Sandviken Airport;4;Sweden;60.593299865722656,16.951400756835938,1500;OurAirports;GVX
+Linköping City Airport;4;Sweden;58.4062004089,15.680500030500001,1500;OurAirports;LPI
+Norrköping Airport;4;Sweden;58.586299896240234,16.250600814819336,1500;OurAirports;NRK
+Eskilstuna Airport;4;Sweden;59.35110092163086,16.70840072631836,1500;OurAirports;\N
+Visby Airport;4;Sweden;57.662799835205,18.346200942993,1500;OurAirports;VBY
+Kalixfors Airport;4;Sweden;67.76480102539062,20.257200241088867,1500;OurAirports;\N
+Spangdahlem Air Base;4;Germany;49.9726982117,6.69250011444,1500;OurAirports;SPM
+Ramstein Air Base;4;Germany;49.4369010925293,7.600279808044434,1500;OurAirports;RMS
+[Duplicate] Giebelstadt Army Air Field;4;Germany;49.648101806599996,9.966489791870002,1500;OurAirports;GHF
+Bückeburg Air Base;4;Germany;52.2785,9.08217,1500;OurAirports;\N
+Celle Airport;4;Germany;52.59120178222656,10.022100448608398,1500;OurAirports;ZCN
+Rheine Bentlage Air Base;4;Germany;52.291199,7.387,1500;OurAirports;\N
+Fritzlar Airport;4;Germany;51.1146,9.286,1500;OurAirports;\N
+Laupheim Air Base;4;Germany;48.220299,9.91002,1500;OurAirports;\N
+Mendig Airport;4;Germany;50.3660011292,7.31533002853,1500;OurAirports;\N
+Niederstetten Army Air Base;4;Germany;49.391945,9.958889,1500;OurAirports;\N
+Roth Airport;4;Germany;49.217498779296875,11.100199699401855,1500;OurAirports;\N
+Fassberg Air Base;4;Germany;52.919399,10.185827,1500;OurAirports;\N
+Grafenwohr Army Air Field;4;Germany;49.698699951171875,11.940199851989746,1500;OurAirports;\N
+Hanau Army Air Field;4;Germany;50.169201,8.96159,1500;OurAirports;ZNF
+Hohenfels Army Air Field;4;Germany;49.218101501464844,11.836099624633789,1500;OurAirports;\N
+Kitzingen Army Air Field;4;Germany;49.743099212646484,10.200599670410156,1500;OurAirports;\N
+Nordholz Naval Airbase;4;Germany;53.7677001953,8.658499717709999,1500;OurAirports;\N
+Diepholz Air Base;4;Germany;52.585556,8.342222,1500;OurAirports;\N
+Geilenkirchen Air Base;4;Germany;50.9608,6.04242,1500;OurAirports;GKE
+Hohn Airport;4;Germany;54.312198638916016,9.538169860839844,1500;OurAirports;\N
+Jever Air Base;4;Germany;53.533501,7.88867,1500;OurAirpor
```

Comment:
I think a bit weird to have "Airport" (in English) in every name. Isn't a local name available?

Reply:
I don't speak the Finnish language; if I'm looking for the Enontekiö Airport, I would definitly search for "Enontekiö Airport" and not for "Enontekiön lentoasema" (which is the local name).

This test file is about internationalization handling.

Reply:
I wonder if we should tag it in some way then, given it's sort of "typed" ("rue de Rivoli" vs "Rivoli Street").
Or maybe adding `lang=en`.
Not a big deal though, and given we are under `world` we may merge as is and see later if it's an issue for some workflow.

Reply:
you're right. I've added the lang column.